wfsim/core.py

```diff
@@ -608,9 +646,10 @@ def _luminescence_timings_simple(n, dG, E0, r, dr, rr, alpha, uE, p, n_electron,
             ne = n_electron[i]
             dt = dr / (alpha * E0[i] * rr)
```

Comment:
I am trying to follow the code here: E0 should be the constant electric field where the anode contribution is not dominating, dr the z step and rr the inverse of the position (constant if outside the anode-dominated field). If in the constant field region, isn't rr changing the value of the field, as it would be clipped to 1 / rA?

Reply:
I don't remember doing this, but during the copy from fax to here, I merged a factor of rA into E0. So now this E0 equals to E (constant between rL and rA) * rA. And thus E0 * rr at inverse rA is that constant.

Reply:
Ah ok, then it makes sense